starters/nuxt3-pinia-vuetify/components/CounterExample/CounterExample.test.ts

```diff
@@ -0,0 +1,78 @@
+import { describe, expect, test } from 'vitest';
+import { createTestingPinia } from '@pinia/testing';
+import CounterExample from './CounterExample.vue';
+import { fireEvent, render, screen } from '@/test/utils';
+
+describe('<CounterExample />', () => {
+	test('Should increase by 1 the count when clicking increment button', async () => {
+		// Test Mock
+		createTestingPinia({
+			initialState: {
+				counterStore: {
+					counter: 0,
+				},
+			},
+		});
+
+		// Act
+		render(CounterExample);
+		const button = screen.getByTestId('increment-button');
+		const countValueCoomponent = screen.getByTestId('count-value');
+
+		// Assertions
+		expect(countValueCoomponent.textContent).toBe('0');
+
+		await fireEvent.click(button);
```

Comment:
I believe the testing-library [recommends](https://testing-library.com/docs/user-event/intro) using the [user-event](https://github.com/testing-library/user-event) module for most user-interactions and falling back to `fireEvent` for things `user-event` can't do

Reply:
I added the dependency in the project, lets focus on changing this when working on issue #479 - In order to unblock the rest of issues